### PR TITLE
RWA-1129-migration parameters have been added into initiation and termination jobs

### PR DESCRIPTION
**Pull request by serdaroguzhanakin**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RWA-1129

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by dharmendrak — APPROVED** [review]

File: k8s/namespaces/wa/wa-task-batch-initiation/aat-01.yaml [hunk 1]
```diff
@@ -6,6 +6,8 @@ spec:
   values:
     job:
       schedule: "2,6,10,14,18,22,26,30,34,38,42,46,50,54,58 * * * *"
+      #Migration works 40. minutes only
```
Comment (mkayad) [inline k8s/namespaces/wa/wa-task-batch-initiation/aat-01.yaml:9]: Do we keep this comment?

File: k8s/namespaces/wa/wa-task-batch-termination/aat-00.yaml [hunk 2]
```diff
@@ -6,6 +6,8 @@ spec:
   values:
     job:
       schedule: "0,4,8,12,16,20,24,28,32,36,40,44,48,52,56 * * * *"
+      #Migration works 0. and 20. minutes
+      #      schedule: "0,20 * * * *"
```
Comment (mkayad) [inline k8s/namespaces/wa/wa-task-batch-termination/aat-00.yaml:10]: same as before

Comment (serdaroguzhanakin) [reply k8s/namespaces/wa/wa-task-batch-termination/aat-00.yaml:10]: let me add an explanation migration time

**Review by mkayad — APPROVED** [review]
LGTM

File: k8s/namespaces/wa/wa-task-batch-initiation/aat-01.yaml (hunk 1 above)
Comment (serdaroguzhanakin) [reply k8s/namespaces/wa/wa-task-batch-initiation/aat-01.yaml]: yeah

**Pull request merged**